server/src/main/java/com/thoughtworks/go/server/service/plugins/processor/console/v1/ConsoleLogAppendRequest1_0.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.thoughtworks.go.server.service.plugins.processor.console.v1;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.annotations.Expose;
+import com.thoughtworks.go.domain.JobIdentifier;
+import com.thoughtworks.go.server.service.plugins.processor.console.ConsoleLogAppendRequest;
+
+public class ConsoleLogAppendRequest1_0 implements ConsoleLogAppendRequest {
+    private static final Gson GSON = new GsonBuilder().excludeFieldsWithoutExposeAnnotation().create();
+
+    @Expose
+    private String pipelineName;
+
+    @Expose
+    private Integer pipelineCounter;
+
+    @Expose
+    private String pipelineLabel;
```

Comment:
@arvindsv -- Can you help me understand the need for `pipelineLabel`?
IMO, pipeline-name and pipeline-counter should be enough to identify pipeline instance.

Reply:
I think I just added it as it was available in the `JobIdentifier` object. But, that's not a good reason. :)

Reply:
Oh, also: `ConsoleService#updateConsoleLog` needs a full `JobIdentifier`.

Reply:
Removed in: https://github.com/gocd/gocd/pull/6397

I've verified that it works [without the pipeline label](https://github.com/arvindsv/docker-elastic-agents/commit/a1e4afaa47f8a79611d06c8f5a46d0797a382dae#diff-73009dc8ddd5490d42a9a60506d6bbe7R98).

Reply:
Thanks :)